{"message":"Self thrown error","stack":"Error: Self thrown error\n    at /home/rizwantariq/My Projects/rosu/routes/genres.js:11:9\n    at newFn (/home/rizwantariq/My Projects/rosu/node_modules/express-async-errors/index.js:16:20)\n    at Layer.handle [as handle_request] (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/layer.js:95:5)\n    at next (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/route.js:137:13)\n    at Route.dispatch (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/route.js:112:3)\n    at newFn (/home/rizwantariq/My Projects/rosu/node_modules/express-async-errors/index.js:16:20)\n    at Layer.handle [as handle_request] (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/layer.js:95:5)\n    at /home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:281:22\n    at Function.process_params (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:335:12)\n    at next (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:275:10)\n    at Function.handle (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:174:3)\n    at router (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:47:12)\n    at newFn (/home/rizwantariq/My Projects/rosu/node_modules/express-async-errors/index.js:16:20)\n    at Layer.handle [as handle_request] (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/layer.js:95:5)\n    at trim_prefix (/home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:317:13)\n    at /home/rizwantariq/My Projects/rosu/node_modules/express/lib/router/index.js:284:7","level":"error","timestamp":"2020-02-24T19:31:26.896Z"}
